swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt

```diff
@@ -219,17 +211,22 @@ private fun emulatorBinary(args: Commands.Start): String =
             emulator
         }
 
+private val usedPorts: MutableList<Int> = mutableListOf()
```

Comment:
This is scary change, I actually used something like this in the beggining, but then figured that it breaks when something else starts emulators outside of Swarmer

Reply:
Ok I might be wrong here, if this is just the used ports it should be fine

Reply:
I think there is always a small risk if something starts emulators outside of Swarmer that can't be avoided. If a emulator would start after `findAvailablePortsForNewEmulator` but before Swarmer starts the emulator. Not sure if there is a way to avoid possible port conflicts in that case. But that is a small window so I think we should be fine.